### ISSUE TITLE
Provide possibility to change OBX-5 type

### ISSUE THREAD

**Issue by unixoid** [krasserm/ipf#38]
For the issue described on https://groups.google.com/group/ipf-user/browse_thread/thread/e4dc776842fc39e5?hl=de the following solution is proposed: add the following method to SegmentAdapter.groovy: 

```
void setObx5Type(String type, int desiredRepetitionsCount = 1) {
    if (! segment.getClass().name.endsWith('.OBX')) {
        throw new AdapterException('only OBX segments can be served by this method')
    }

    for (int i = 0; i < desiredRepetitionsCount - count(5); ++i) {
        nrp(5)
    }

    putAt(2, type)
    Varies.fixOBX5(target, target.message.parser.factory)
}
```

**Comment by unixoid** [comment]
NB: according to HAPI model classes, the "varies" data type is being used not only in OBX segments, but as well in MFA, MFE, QPD, and RDT.  The question is whether they all matter.

**Comment by unixoid** [comment]
Default type of OBX-5 can be set as Java system property Varies.DEFAULT_OBX2_TYPE_PROP

**Comment by unixoid** [comment]
Possibility to change OBX-5 type; closed by 03e81de1113df7ad284b75c0801441c185fe76b9
